src/Services/Search/SearchFieldComparableValue.php

```diff
@@ -16,22 +16,19 @@
 
 class SearchFieldComparableValue
 {
-    /**
-     * @var mixed
-     */
-    private $value;
-
     /**
      * @var string
      */
     private $comparator;
```

Comment:
weird that it did not incorporate comparator below since is set (some weird lines before it but seems like as long as they have `$this->comparator`, then should be fine to bring it in

Reply:
I think this was a case of rector being somewhat conservative, but I did it manually.